specification/datashare/resource-manager/Microsoft.DataShare/preview/2018-11-01-preview/DataShare.json

```diff
@@ -4525,7 +4544,8 @@
     "ScheduledSynchronizationSetting": {
       "description": "A type of synchronization setting based on schedule",
       "required": [
-        "properties"
+        "properties",
+        "kind"
```

Comment:
There is no field name "kind" in these properties, like the next others. "x-ms-discriminator-value" is not in CI error output, just "discriminator" field is in CI error output.

Reply:
> @sarajang yeah, I saw the "discriminator" field should be required in CI output, but the field with "x-ms-discriminator-value" doesn't show in it. Therefore, I am wondering whether this field should also be required?
> @ravbhatnagar could you review again for the change of required field? Thanks!

@ryansbenson  can you please look into this? I think this is how kind and x-ms-discriminator are handled for read only